Giveme5W1H/extractor/extractors/cause_extractor.py

```diff
@@ -129,8 +129,7 @@ def _evaluate_tree(self, tree):
                 sibling = sibling.right_sibling()
 
             # NP-VP-NP pattern found .__repr__()
-	    
-            if sibling.label() == 'VP' and "('NP'" in sibling.__repr__():
+	    if sibling.label() == 'VP' and "('NP'" in sibling.__repr__():
```

Comment:
it appears that the intendation is not correct here. could you please fix this?

Reply:
Sure.